api/analysis/safety.go

```diff
@@ -29,13 +30,21 @@ type SafetyIssue struct {
 }
 
 //SafetyStartAnalysis analyses the output from Safety and sets cResult based on it.
-func SafetyStartAnalysis(CID string, cOutput string) {
+func SafetyStartAnalysis(CID string, cOutput string, RID string) {
 
 	var outputJSON string
+	var outputWarnings []string
+
 	analysisQuery := map[string]interface{}{"containers.CID": CID}
 
-	errorSafety := strings.Contains(cOutput, "ERROR_RUNNING_SAFETY")
-	if errorSafety {
+	reqNotFound := strings.Contains(cOutput, "ERROR_REQ_NOT_FOUND")
```

Comment:
This standardised validation can be done at all similar functions.

Reply:
Nice! I have added this validation in some commits after this one! 🙃